E3Next/Settings/BaseSettings.cs

```diff
@@ -367,6 +367,38 @@ public static void LoadKeyData(string sectionKey, string Key, IniData parsedData
                 }
             }
         }
+
+        public static void LoadKeyData(string sectionKey, IniData parsedData, Dictionary<string, List<Data.Spell>> collectionToAddTo) {
```

Comment:
Added new LoadKeyData, since the usage of this is to load a INI Section without a specific subsection key.

I.e.: 
	    LoadKeyData("Burn", ParsedData, BurnsDictionary);
vs
            LoadKeyData("Burn", "Quick Burn", ParsedData, QuickBurns);